HW8/marian_hrychak/task_3/main.py

```
@@ -0,0 +1,18 @@
+from math_utils import *
+
+question = input("Which figure do you want to calculate the area of? : ")
+
+match question.lower():
+    case "rectangle":
+        length_answer = float(input("Enter the length: "))
+        width_answer = float(input("Enter the width: "))
+        print(f"Area of the rectangle is {rectangle(length_answer, width_answer)}")
```

Ruff (0.8.2)

`rectangle` may be undefined, or defined from star imports

(F405)